src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileUpdatorTest.java

```diff
@@ -195,14 +228,45 @@ public void deleteRoles_InternalServerError() throws Exception {
         ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         String body = mapper.writeValueAsString(userProfileRolesResponse);
 
+        Response responseMock = mock(Response.class);
+        when(responseMock.status()).thenReturn(500);
+
+        when(userProfileRepositoryMock.findByIdamId(any(String.class))).thenReturn(Optional.ofNullable(userProfile));
+        when(idamFeignClientMock.deleteUserRole("1234", "pui-case-manager")).thenReturn(responseMock);
+
+        UserProfileRolesResponse response = sut.updateRoles(updateUserProfileData, userProfile.getIdamId());
+        assertThat(response.getRoleDeletionResponse().get(0).getIdamStatusCode()).isEqualTo("500");
+
+        verify(userProfileRepositoryMock, times(1)).findByIdamId(any(String.class));
+        verify(idamFeignClientMock, times(1)).deleteUserRole(any(String.class), any(String.class));
+        verify(responseMock, times(1)).close();
+    }
+
+    @Test
+    public void deleteRoles_FeignException() {
+        RoleName roleName1 = new RoleName("pui-case-manager");
+        Set<RoleName> roles = new HashSet<>();
+        roles.add(roleName1);
+        updateUserProfileData.setRolesDelete(roles);
```

Comment:
prefix test name with test

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/385/commits/04d04c3144cda8c41c2c8b875f3a62b0fa855501